cg/clients/janus/api.py

```diff
@@ -0,0 +1,28 @@
+"""The JanusAPIClient."""
+
+from http import HTTPStatus
+
+import requests
+from requests import Response
+
+from cg.clients.janus.dto.create_qc_metrics_request import CreateQCMetricsRequest
+from cg.clients.janus.exceptions import JanusClientError, JanusServerError
+
+
+class JanusAPIClient:
+    def __init__(self, config: dict[str]):
+        self.host = config["janus"]["host"]
+
+    def qc_metrics(self, collect_qc_request: CreateQCMetricsRequest) -> dict:
+        endpoint: str = f"{self.host}/collect_qc_metrics"
+        response = requests.post(endpoint, data=collect_qc_request.model_dump_json())
```

Comment:
Function calls are easier to read when they do not contain other nested function calls. Consider extracting the `collect_qc_request.model_dump_json()` to a separate variable